src/v0/destinations/google_adwords_enhanced_conversions/transform.js

```diff
@@ -45,8 +45,9 @@ const responseBuilder = async (metadata, message, { Config }, payload) => {
   let { customerId, loginCustomerId } = Config;
   const { configData } = Config;
 
+  let configDetails = {};
```

Comment:
why `let` --> you are not updating the variable twice

Reply:
earlier was reassigning it so used let, now using ternary condition and const